kong/plugins/moesif/handler.lua

```diff
@@ -55,7 +55,7 @@ function MoesifLogHandler:access(conf)
   local mimetype = nil
   local content_length = headers["content-length"]
 
-  if tonumber(content_length) <= conf.max_body_sime_limit then 
```

Comment:
i mean if content-length header is not set, you can't guarantee that body size < 100K.. in theory i would want to ignore body if content-length is nil. do u think otherwise?

Reply:
Definitely agree. People get this wrong all the time especially request side (even though not to spec)